debian/tests/control

```diff
@@ -1,7 +1,7 @@
-Tests: libmutter-7-dev
+Tests: libmutter-6-dev
```

Comment:
I guess this should be libmutter-7 instead of 6 and also there is a file debian/tests/libmutter-6-dev that should be renamed.

Reply:
Actually instead of renaming the libmutter-6-dev file we should probably get the libmutter-7-dev from debian that got dropped here.

Reply:
This is what's in Debian upstream, as incredible as it may sound. I can follow up on Salsa, but wouldn't consider this a blocker.

Reply:
Oh, that is weird and definitely wrong (unless it is testing an upgrade path from libmutter-6 :P) - Ok, if it is the same on debian I would follow-up there and we can get the fix once we update again - considering this doesn't break the pkg build.